index.js

```diff
@@ -0,0 +1,60 @@
+document.addEventListener('DOMContentLoaded', () => {
+// console.log("dom")
+setupDroplist();
+fetchMovieList()
+})
+// add list to drop list
+const setupDroplist = ()=>{
+  const getMovieList = document.querySelector('#droplist')
+  getMovieList.addEventListener('click', fetchMovieList)
+  console.log( "click")
+}
```

Comment:
index.js:1-11. This all looks like a great start. Setting an event listener for when the DOM loads to then setup the dropdown list and fetch all the movie titles.  The only problem is what I mentioned above.